### PR TITLE
Feat: bulk select component

### PR DESCRIPTION
**Pull request by rkpattnaik780**
### Type of change

- Visual component

### Description

Bulk select component for tables along with its unit tests.

### Screenshot
Screenshot from 2020-05-08 17-18-38

### CODE REVIEW

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.test.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import { BulkSelect, IBulkSelectProps } from "./BulkSelect";
+import { DropdownItem } from "@patternfly/react-core";
+
+describe("<BulkDelete />", () => {
+  const items: any[] = [
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/components/BulkSelect/BulkSelect.test.tsx:12]: please avoid any type

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  Dropdown,
+  DropdownToggle,
+  DropdownToggleCheckbox
+} from "@patternfly/react-core";
+
+export interface IBulkSelectProps {
+  isChecked: boolean;
+  isOpen: boolean;
+  handleOnToggle: (val: boolean) => void;
+  items: any[];
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx:17]: avoid any type

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      onSelect={handleOnSelect}
+      toggle={
+        <DropdownToggle
+          id="bulk-select-dropdown-toggle"
+          splitButtonItems={[
+            <DropdownToggleCheckbox
+              id="bulk-select-checkbox"
+              key="bulk-select-checkbox"
+              aria-label="Select all items"
+              onChange={checked => handleOnChange(checked)}
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx:44]: no need to pass arrow function.
onChange={handleOnChange} will work

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+              isChecked={isChecked}
+            >
+              {selectedCount} selected
+            </DropdownToggleCheckbox>
+          ]}
+          onToggle={value => handleOnToggle(value)}
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx:50]: No need to pass arrow function
onToggle={handleOnToggle} will work

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx (hunk 2 above)
Comment (rkpattnaik780) [reply console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx]: Replaced with `React.ReactNode[]` now.

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx (hunk 4 above)
Comment (rkpattnaik780) [reply console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx]: Modified!

**Pull request merged**